src/AppBundle/Controller/Admin/StatisticsController.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace AppBundle\Controller\Admin;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Request;
+
+/**
+ * @Route("admin/statistics")
+ */
+class StatisticsController extends Controller
+{
+    private $months = [
+        "01" => "Januari",
+        "02" => "Februari",
+        "03" => "Mars",
+        "04" => "April",
+        "05" => "Maj",
+        "06" => "Juni",
+        "07" => "Juli",
+        "08" => "Augusti",
+        "09" => "September",
+        "10" => "Oktober",
+        "11" => "November",
+        "12" => "December"
+    ];
+
+    /**
+     * @Route("/", name="admin_statistics")
+     */
+    public function indexAction(Request $request)
+    {
+        $city = $request->query->get("city", 2);
```

Comment:
Please don't hard code value 2. Why id 2? Can you go for `$cityRepository->findAll()[0]`?